ouroboros-network/src/Ouroboros/Network/Node.hs

```diff
@@ -32,9 +32,10 @@ import           Network.Mux.Channel
 import           Network.Mux.Time
 
 import           Ouroboros.Network.Block
-import           Ouroboros.Network.Chain (Chain (..), Point)
-import qualified Ouroboros.Network.Chain as Chain
-import           Ouroboros.Network.ChainProducerState (ChainProducerState (..),
+-- TODO Should this be impored here
```

Comment:
Can somebody from the network team (@coot) clarify why `Chain` is used here? I this also a test/mock/example module? If so, then the name (and/or module doc) should reflect it.

Reply:
@mrBliss indeed, that is our initial naive node implementation. It even pre-dates `typed-protocols`.